### ISSUE TITLE
Proposal: Update link to discussion about Reuters Dataset topic labels

### ISSUE THREAD

**Issue by SteffenBauer** [keras-team/keras-io#134]
The section about the Reuters dataset at keras.io ( https://keras.io/api/datasets/reuters/ ) contains a link to this topic discussion at the Keras repository, where I was providing the results of my investigation into the topic label mapping:

https://github.com/keras-team/keras/issues/12072

As the question about the label mapping pops up occasionally, and I found that the results are now even used in some publications, I collected all the code and data from the issue keras-team/keras#12072 here, and I'd like to propose to update the link at keras.io accordingly:

https://github.com/SteffenBauer/KerasTools/tree/master/Reuters_Analysis
